js/source/tile.js

```diff
@@ -20,11 +21,10 @@ function Tile(coord, size, sourceMaxZoom) {
     this.uses = 0;
     this.tileSize = size;
     this.sourceMaxZoom = sourceMaxZoom;
+    this.pixelRatio = EXTENT / size;
```

Comment:
What's this used for? Is it expected that standard 4096 extent and 512 tile gives a 8.0 "pixel ratio"?